# Request

# Target pose
geometry_msgs/PoseStamped pose

---
# Response

bool success
